Balance/Home.swift

```diff
@@ -181,11 +182,11 @@ struct HomeView: View {
 }
 
 #if DEBUG
-struct MainView_Previews: PreviewProvider {
+struct Home_Previews: PreviewProvider {
     static var previews: some View {
         HomeView()
-            .environmentObject(BalanceScheduler())
-            .environmentObject(MockDataStorageProvider())
+            .environmentObject(Account(accountServices: []))
+            .environmentObject(FirebaseAccountConfiguration<FHIR>(emulatorSettings: (host: "localhost", port: 9099)))
```

Comment:
@gperissetcelteeka is there a way we can configure so we don't have to hardcode `localhost` and port?